Sources/XCTest/Public/XCTestMain.swift

```diff
@@ -87,6 +87,33 @@ public func XCTMain(_ testCases: [XCTestCaseEntry]) -> Never {
     case .list(type: .json):
         TestListing(testSuite: rootTestSuite).printTestJSON()
         exit(0)
+    case let .help(invalidOption):
+        if let invalid = invalidOption {
+            print("Error: Invalid option \"\(invalid)\"\n")
+        }
+        let exeName = CommandLine.arguments[0].lastPathComponent
+        let sampleTest = rootTestSuite.list().first ?? "Tests.FooTestCase/testFoo"
+        let sampleTests = sampleTest.prefix(while: { $0 != "/" })
+        print("""
+              Usage: \(exeName) [OPTION]
+                     \(exeName) [TESTCASE]
+              Run and report results of test cases.
+
+              With no OPTION or TESTCASE, runs all test cases.
+
+              -l, --list-test              List tests line by line to standard output
```

Comment:
Could we add a header `OPTIONS:` here above this section containing all the options, since the section after this has a header (`Examples:`)? And maybe it would help to phrase use `TESTCASE Examples:` as the title of the latter header, to reinforce the "OPTIONS or TESTCASE" terms above?